shared-bindings/pwmio/PWMOut.c

```diff
@@ -46,23 +46,23 @@ void common_hal_pwmio_pwmout_raise_error(pwmout_result_t result) {
             mp_arg_error_invalid(MP_QSTR_frequency);
             break;
         case PWMOUT_INVALID_FREQUENCY_ON_PIN:
-            mp_raise_ValueError(MP_ERROR_TEXT("Frequency must match existing PWMOut using this timer"));
+            mp_arg_error_invalid(MP_QSTR_frequency);
             break;
         case PWMOUT_VARIABLE_FREQUENCY_NOT_AVAILABLE:
-            mp_raise_ValueError(MP_ERROR_TEXT("Cannot vary frequency on a timer that is already in use"));
+            mp_arg_error_invalid(MP_QSTR_variable_frequency);
             break;
         case PWMOUT_ALL_TIMERS_ON_PIN_IN_USE:
-            mp_raise_ValueError(MP_ERROR_TEXT("All timers for this pin are in use"));
+            mp_raise_RuntimeError(MP_ERROR_TEXT("Internal resource(s) in use"));
             break;
         case PWMOUT_ALL_TIMERS_IN_USE:
-            mp_raise_RuntimeError(MP_ERROR_TEXT("All timers in use"));
+            mp_raise_RuntimeError(MP_ERROR_TEXT("Internal resource(s) in use"));
             break;
         case PWMOUT_ALL_CHANNELS_IN_USE:
-            mp_raise_RuntimeError(MP_ERROR_TEXT("All channels in use"));
+            mp_raise_RuntimeError(MP_ERROR_TEXT("Internal resource(s) in use"));
             break;
         default:
```

Comment:
suggest merging these *constants* since I think there was nothing but error message generation that cared about the distinction.

Reply:
Done since it makes sense here.